### DISCUSSION TITLE
blog/dimensionality-reduction-using-auto-encoders

### DISCUSSION THREAD

**Discussion by giscus** [PrabhuKiran8790/prabhukirankonda#3] [General]
# blog/dimensionality-reduction-using-auto-encoders

Let's look at how we can perform the dimensionality reduction using the Auto-Encoders using Tensorflow

https://prabhukirankonda.vercel.app/blog/dimensionality-reduction-using-auto-encoders

**Comment by PrabhuKiran8790** [comment]
This is a test comment

**Reply by PrabhuKiran8790** [reply]
```py
print("also a test comment")
```

**Comment by warrenbeufesche** [comment]
I am glad to see you have comments enabled. I am sorry I do not have a pertinent comment on your post though.

**Reply by PrabhuKiran8790** [reply]
that's fine. No worries
